core/tx/textual/valuerenderer/valuerenderer_test.go

```diff
@@ -0,0 +1,144 @@
+package valuerenderer_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/reflect/protoreflect"
+
+	"cosmossdk.io/core/tx/textual/internal/testpb"
+	"cosmossdk.io/core/tx/textual/valuerenderer"
+	"cosmossdk.io/math"
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
This creates an issue: the `core` go module imports from `cosmos-sdk/types`, which shouldn't be allowed.

Any ideas how to fix this (before we move Dec to `math`)? For now I guess I can skip decimal tests.

Reply:
out of curiosity why is this in core? would we bring proto message signing into core as well?

Reply:
I think it should be in a tx module instead of core personally